lib/bundles/codemirror.js

```diff
@@ -34,12 +34,29 @@
 // Codemirror base lib
 
 import {EditorView, basicSetup} from "codemirror";
-import {css} from "@codemirror/lang-css"
+import {css} from "@codemirror/lang-css";
+import {json} from "@codemirror/lang-json";
+import {sql} from "@codemirror/lang-sql";
+import {highlightTree} from "@lezer/highlight";
+import {defaultHighlightStyle} from "@codemirror/language";
```

Comment:
`@codemirror/language` should be added to our dependencies if we use in our code.